.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+### Pull request name 
+
+While your working on your PR please add the [WIP] indication in the title, to avoid reviewing before it's ready. Also, please try to have a descriptive title.
```

Comment:
In my opinion, a PR should only be opened once the issue is solved. Therefore I think that this field is not necessary. What do you guys think @GullinBustin @WoodElv?

Reply:
Interesting. I used to create a PR just when the branch is created so we can start a discussion about the changes, or some collaborators can start reviewing even the PR is not finished. In fact, github has a [draft tag](https://github.blog/2019-02-14-introducing-draft-pull-requests/) that we could use to do this. Also, the checklist at the template illustrates how advanced is the PR.

Reply:
Agreed, @GullinBustin @WoodElv any thoughts?

Reply:
Yes, I think it's a good idea the using of PR in draft state.